airbyte-integrations/connectors/destination-postgres/src/main/kotlin/io/airbyte/integrations/destination/postgres/schema/PostgresColumnManager.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.postgres.schema
+
+import io.airbyte.cdk.load.message.Meta
+import io.airbyte.cdk.load.schema.model.ColumnSchema
+import io.airbyte.integrations.destination.postgres.spec.PostgresConfiguration
+import jakarta.inject.Singleton
+
+/**
+ * Manages column names and ordering for Postgres tables based on whether legacy raw tables mode is
+ * enabled.
+ */
+@Singleton
+class PostgresColumnManager(
+    private val config: PostgresConfiguration,
+) {
```

Comment:
Just FYI in the future I want to build this into the CDK so we don't have to implement this, but for now this is the way.